clib/src/main/resources/scala-native/atomic.cpp

```diff
@@ -0,0 +1,13 @@
+#include <atomic>
+
+// Due to some bug (in GCC?) it might not be possible to access
```

Comment:
I may be missing something but is this GCC or glibc?

Reply:
To be honest I don't remember what was the original cause, I was having these problems ~ 1-1.5 years ago. I've tried again to move these 2 definitions to `.c` file and it worked, so I've removed `atomic.cpp` entirely.